x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/DriverStatusTests.java

```diff
@@ -52,7 +52,7 @@ public void testToXContent() {
         assertThat(Strings.toString(status, true, true), equalTo("""
             {
               "session_id" : "ABC:123",
-              "task_description" : "test",
+              "description" : "test",
```

Comment:
I wonder if this (along with https://github.com/elastic/elasticsearch/pull/123457 dependency) worth backporting to 9.0

Reply:
I think it is! And, potentially, to 8.x.

If we start building tooling based on profiles, it'd be nice if that worked also on older branches without too many adjustments. It also means that as humans, the profiles remain easier to compare and have less surprises when e.g. users provide a profile that's from an older version.

Reply:
https://github.com/elastic/elasticsearch/pull/123863